.github/workflows/run_tests.yaml

```diff
@@ -77,4 +77,26 @@ jobs:
         if: always()
         with:
           name: cromwell_logs
-          path: cromwell_logs/
\ No newline at end of file
+          path: cromwell_logs/
+
+  python:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.9'
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+      - name: Lint with Ruff
+        run: |
+          pip install ruff
+          ruff check --output-format=github .
+      - name: ScoreBGE Unit tests
+        run: |
```

Comment:
Does this mean that linting + testing this particular test are wrapped up in the same GH action? It might be worth splitting them just for conceptual clarity.

Reply:
good call, I will split them up